src/article/JATS4R.rng

```diff
@@ -315,6 +315,14 @@
     </element>
   </define>
 
+  <define name="fn">
+    <element name="fn">
+      <oneOrMore>
+        <ref name="just-para.class"/>
```

Comment:
Also here please use the explicit tag instead of just-para.class, for better readability.

```
<ref name="p"/>
```